vendor.go

```diff
@@ -191,7 +206,7 @@ func addCleanup(c func()) {
 
 func abortonerr(err error, details string) {
 	if err != nil {
-		fmt.Sprintf("unexpected error[%s] %s\n", err, details)
+		fmt.Printf("unexpected error[%s] %s\n", err, details)
```

Comment:
God my code is stupid x_x. Sorry for that. But I did develop this in like a few hours and with no review =P.